flixel/tweens/FlxTween.hx

```diff
@@ -523,6 +567,39 @@ class FlxTween implements IFlxDestroyable
 		active = false;
 		_running = false;
 		finished = true;
+		
+		if (_thens != null && _thens.length > 0)
+		{
+			var then = _thens[0];
+			
+			_thens.splice(0, 1);
+			
+			if (then.delay <= 0)
```

Comment:
This looks like it might fail with `wait(0)` (which should just be "no wait" - doesn't make sense if written like that but the delay could be dynamically set from some variable that happens to become 0 at some point)? And what's the expected result for negative values?